ts/features/itwallet/common/utils/itwClaimsUtils.ts

```diff
@@ -410,3 +411,8 @@ const FISCAL_CODE_REGEX =
  */
 export const extractFiscalCode = (s: string) =>
   pipe(s.match(FISCAL_CODE_REGEX), match => O.fromNullable(match?.[0]));
+
+/**
+ * Truncate long strings to avoid performance issues when rendering claims.
+ */
+export const getSafeText = (text: string) => truncate(text, { length: 128 });
```

Comment:
I think we should add something to indicate that the values ​​have been truncated, maybe an ellipsis at the end. What do you think?

Reply:
I agree, and it's automatically done by the `truncate` function. You can try to set a length like 4.